feature/community/src/main/java/com/iteneum/community/data/CommunityViewModel.kt

```diff
@@ -0,0 +1,36 @@
+package com.iteneum.community.data
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.*
+import androidx.compose.runtime.State
+import androidx.compose.runtime.mutableStateListOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.snapshots.SnapshotStateList
+import androidx.lifecycle.ViewModel
+import com.iteneum.community.domain.CardMenuItem
+import com.iteneum.community.domain.CardModel
+
+class CommunityViewModel : ViewModel() {
+    private val _communityViewModelState = mutableStateOf(CommunityViewStates())
+    val communityViewModelState: State<CommunityViewStates>
+        get() = _communityViewModelState
+    private val _buttonCards = mutableStateListOf<CardModel>()
```

Comment:
Please be more descriptive of what kind of buttons or cards are

Reply:
will do,   do you have any suggestions for a variable that is not exposed?

Reply:
please elaborate because this is not a button or a card, this variable is both a "ButtonCard",  but does the project has a guide for naming variables if so, please forward it to my mail address to fix it